src/solvespace.h

```diff
@@ -92,10 +92,11 @@ __attribute__((__format__ (__printf__, 1, 2)))
 #endif
 std::string ssprintf(const char *fmt, ...);
 
-inline int WRAP(int v, int n) {
+template<typename T>
+inline T WRAP(T v, T n) {
```

Comment:
(I think you missed my earlier question.) I don't understand this change--what's the point of making WRAP generic if it's only used with ints anyway *and* you need to cast it each time?

Reply:
Ah, I did miss your question. To be honest, I have no idea, that's the risk of me leaving commits un-submitted for so long.